python/taichi/lang/impl.py

```diff
@@ -286,15 +286,21 @@ def indices(*x):
 index = indices
 
 
-def static(x):
+def static(x, *xs):
+  if len(xs): # for python-ish pointer assign: x, y = ti.static(y, x)
+    return [static(x)] + [static(x) for x in xs]
```

Comment:
Sorry for breaking your ccbb81a :) I thought I've found a more elegant way to deal with it like other function does.
I mean, to use python built-in argument number dismatch error like `ti.grouped` will do.
If you don't like this, pls feel free to revert it as you want.

Reply:
Not sure if I get your point correctly

Reply:
I mean, `ti.static(x, *xs)` have the same error message with the `ti.static(x)` instead a custom ValueError.

Reply:
I see. But keep in mind that having recursion here will lead to confusing stack backtrace if the final entry of `ti.static` leads to any error...

Reply:
Also pls think if we want `ti.static(range(2), range(3))` as syntax correct.

Reply:
This is really a corner case - I would say the users should avoid confusing expressions like these. Let's allow this for now - the users should use it at their own risk.